setup.py

```diff
@@ -14,6 +14,7 @@
         #"jmbo>=3.0.0", uncomment when released
         #"beautifulsoup4",
         #"pypandoc",
+        "django-simplemde",
```

Comment:
Just a note we prefer requirements.pip vs setup.py for pinning versions and requirements.